<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loading</class>
 <widget class="QDialog" name="Loading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lade Optionen</string>
  </property>
  <widget class="QPushButton" name="Button_Keep_Program">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>42</y>
     <width>179</width>
     <height>95</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Reject_Program">
   <property name="geometry">
    <rect>
     <x>181</x>
     <y>42</y>
     <width>181</width>
     <height>95</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Abort">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>139</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Question_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Was soll mit dem aktuellen Programm passieren?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_Abort</sender>
   <signal>clicked()</signal>
   <receiver>Loading</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
